pytest-selenium/pages/content.py

```diff
@@ -50,6 +78,10 @@ class BookBanner(Region):
         _chapter_title_locator = (By.CSS_SELECTOR, "div > h1 > span.os-text")
         _chapter_section_locator = (By.CSS_SELECTOR, "div > h1 > span.os-number")
 
+        @property
+        def book_title_element(self):
```

Comment:
didn't mike add a bookbanner region in #188 ? can't we use that here?

Reply:
oh i think that's left over from a different thing i was trying. ooops.